translations/translation_de.xml

```diff
@@ -89,18 +89,18 @@
         <text name="msg_welcome" text="Willkommen zum Seasons Mod by Realismus Modding. Benötigst Du Hilfe oder eine Erklärung nutze das Hilfemenü (per ESC erreichbar)." />
 
         <!-- Missing translation of "Calendar" -->
-        <text name="ui_pageCalender" text="" />
```

Comment:
Would something like Ekonomie be better? You cant see the prices, but more economical info: the trend of the price over a year. Different from the price table in the esc menu